payplug/resources.py

```diff
@@ -391,3 +391,32 @@ class Operation(APIResource):
         An operation.
         """
         pass
+
+
+class InstallmentPlan(APIResource):
```

Comment:
Je pense qu'on voudrait : 

```suggestion
class InstallmentPlan(APIResource, VerifiableAPIResource, ReconstituableAPIResource):
```

Comme pour les autres ressources d'api.